Core/src/org/sleuthkit/autopsy/ingest/IngestTasksScheduler.java

```diff
@@ -318,6 +318,47 @@ synchronized void cancelPendingTasksForIngestJob(IngestJobPipeline ingestJobPipe
         }
         return topLevelFiles;
     }
+    
+    /**
+     * Schedules file ingest tasks for the ingest manager's file ingest threads.
+     * Files from streaming ingest will be prioritized.
+     */
+    synchronized private void shuffleFileTaskQueues() {
+	try {
+	    shuffleStreamingFileTaskQueues();
+	    shuffleRootFileTaskQueues();
+	} catch (InterruptedException ex) {
+	    IngestTasksScheduler.logger.log(Level.INFO, "Ingest tasks scheduler interrupted while blocked adding a task to the file level ingest task queue", ex);
+	    Thread.currentThread().interrupt();
+	}
+    }
```

Comment:
Name changes we talked about...
shuffleFileTaskQueues => refillIngestThreadQueue
shuffleStreamingFileTaskQueues =>takeFromStreamingTaskQueue
shuffleRootFileTaskQueues =>takeFromBatchTasksQueues